Formula/edirect.rb

```diff
@@ -25,10 +24,11 @@ def install
 
     ENV.prepend_create_path "PERL5LIB", prefix/"perl5/lib/perl5"
     ENV["OPENSSL_PREFIX"] = Formula["openssl"].opt_prefix # for Net::SSLeay
-    pms = %w[Encode::Locale File::Listing HTML::Parser HTML::Tagset HTML::Entities HTML::Tree
+    pms = %w[XML::Simple Encode::Locale File::Listing HTML::Parser HTML::Tagset HTML::Entities HTML::Tree
              HTTP::Cookies HTTP::Date HTTP::Message HTTP::Negotiate LWP::MediaTypes IO::Socket::SSL
              LWP::Protocol::https URI WWW::RobotRules Mozilla::CA Net::SSLeay]
-    system "cpanm", "--self-contained", "-l", prefix/"perl5", *pms
+    system "cpan", "-f", "XML::SAX::Expat"
```

Comment:
```suggestion
    system "cpanm", "--force", "--self-contained", "-l", prefix/"perl5"`, "XML::SAX::Expat"
```
Using `cpan` like this won't add `XML::SAX::Expat` to the built bottle, and so the bottle will be missing this library. Does `cpanm` work?

Reply:
The installation for XML::SAX::Expat would fail when cpanm was used, but worked when I did it with cpan.

Reply:
When installed with `cpan` in this way, it's being installed into a system-wide directory that won't be included in the bottle. Can you troubleshoot locally installation with `cpanm` and see if you can get it to work?

Reply:
That's a useful data point, to know that it does work with `cpan`. I wonder why it doesn't work with `cpanm`…